nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/listener/test/SimpleListenerTest.java

```diff
@@ -40,14 +40,13 @@
 import org.nuxeo.ecm.platform.comment.api.CommentImpl;
 import org.nuxeo.ecm.platform.comment.api.CommentManager;
 import org.nuxeo.ecm.platform.comment.permissions.test.CommentFeature;
-import org.nuxeo.ecm.platform.test.PlatformFeature;
 import org.nuxeo.runtime.api.Framework;
 import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
 import org.nuxeo.runtime.transaction.TransactionHelper;
 
 @RunWith(FeaturesRunner.class)
-@Features({ PlatformFeature.class, CommentFeature.class })
+@Features({ CommentFeature.class })
```

Comment:
No need for curly brackets `@Features({ CommentFeature.class })` -> `@Features(CommentFeature.class )`